delivery_carrier_label_default/README.rst

```diff
@@ -0,0 +1,82 @@
+================================
```

Comment:
May be change Readme to modular like [https://github.com/OCA/maintainer-tools/tree/master/template/module/readme](https://github.com/OCA/maintainer-tools/tree/master/template/module/readme)

Reply:
It was already done, check this commit: 603b0bf88c79a6f31f0138952054ece723a58b9b

Reply:
thanks sorry